test/cmdlineTests/standard_optimizer_yulDetails_optimiserSteps_with_empty_sequence_no_yul/in.sol

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.0;
+
+contract C {
+    function f() public pure {}
+}
```

Comment:
Is this really that much of an issue - I prefer for input *.sol files to be separate, since it's easier to see  their contents. Of course, if there's a good reason, I'll move them to the .json file.

Reply:
It's fine. I also generally prefer them separate, just here it's a bit funny because the file name is almost as long as the content when you get rid the contract. And I suggested removing the contract because it does not add much and CLI tests are already quite verbose for the value they add.